docs/getting-started.md

```diff
@@ -189,35 +187,49 @@ We also need to configure sass-loader to understand the `@material` imports used
 > [Appendix](#appendix-configuring-a-sass-importer-for-nested-node_modules) below on how to configure a custom importer
 > instead.
 
-In order to add vendor-specific styles to the Sass files, we need to configure `autoprefixer` through PostCSS.
+In order to add vendor-specific styles to the Sass files, we need to configure `postcss-preset-env` through PostCSS.
 
 You'll need all of these Node dependencies:
-- [autoprefixer](https://www.npmjs.com/package/autoprefixer): Parses CSS and adds vendor prefixes to CSS rules
-- [postcss-loader](https://github.com/postcss/postcss-loader): Loader for Webpack used in conjunction with autoprefixer
+- [postcss-loader](https://www.npmjs.com/package/postcss-loader): Loader for Webpack used in conjunction with `postcss-preset-env`
+- [postcss-preset-env](https://www.npmjs.com/package/postcss-preset-env): Parses CSS and adds vendor prefixes to CSS rules
 
 You can install all of them by running this command:
 
 ```
-npm install --save-dev autoprefixer postcss-loader
+npm install --save-dev postcss-loader postcss-preset-env
 ```
 
-Add `autoprefixer` at the top of your `webpack.config.js`:
+Now create a new `postcss.config.js` file with the following contents:
 
 ```js
-const autoprefixer = require('autoprefixer');
+module.exports = {
+    plugins: {
+        "postcss-preset-env": {
+            stage: 0,
```

Comment:
postcss-preset-env is unnecessary because it's just a wrapper for Autoprefixer and polyfills according to the Autoprefixer repo. 

> postcss-preset-env is a plugins preset with polyfills and Autoprefixer to write future CSS today.

Reply:
Everything changed in the CSS section should be reverted because there's no need for postcss-preset-env.